hazelcast-tpc-engine/src/test/java/com/hazelcast/internal/tpc/AsyncSocketTest.java

```diff
@@ -44,27 +46,37 @@ public abstract class AsyncSocketTest {
 
     @After
     public void after() throws InterruptedException {
-        for (Eventloop eventloop : eventloops) {
-            eventloop.shutdown();
-            eventloop.awaitTermination(5, TimeUnit.SECONDS);
-        }
+        terminateAll(eventloops);
     }
 
     @Test
     public void test_remoteAddress_whenNotConnected() {
         Eventloop eventloop = createEventloop();
-        AsyncSocket socket = eventloop.openAsyncTcpSocket();
+        AsyncSocket socket = eventloop.openTcpAsyncSocket();
         socket.setReadHandler(mock(ReadHandler.class));
         socket.activate(eventloop);
 
         SocketAddress remoteAddress = socket.getRemoteAddress();
         assertNull(remoteAddress);
     }
 
+
+    @Test
+    public void test_localAddress_whenNotConnected() {
+        Eventloop eventloop = createEventloop();
+        AsyncSocket socket = eventloop.openTcpAsyncSocket();
+        socket.setReadHandler(mock(ReadHandler.class));
+        socket.activate(eventloop);
+
+        SocketAddress localAddress = socket.getLocalAddress();
+        System.out.println(localAddress);
```

Comment:
Better to use logging or remove it if it's just temporary debug

Reply:
Should be removed.